src/tests/ColorPicker.test.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import Enzyme from 'enzyme';
 import { shallow } from 'enzyme';
-import ColorPickers from '../components/ColorPickers';
```

Comment:
You need this line to import the component you are testing :)

Reply:
Thank you! I'm just currently trying to make the test work properly, having some issues with it 🙂